pkgs/applications/misc/polychromatic/default.nix

```diff
@@ -80,4 +82,3 @@ python3Packages.buildPythonApplication rec {
     platforms = platforms.linux;
     maintainers = with maintainers; [ evanjs ];
   };
-}
```

Comment:
Revert this

Reply:
I will fix, it was a miss on my part, I have it on my system